io.github.nemuelw.protodesk.yml

```diff
@@ -0,0 +1,37 @@
+app-id: io.github.nemuelw.protodesk
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+base: io.qt.PySide.BaseApp
+base-version: '6.7'
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+command: protodesk
+
+finish-args:
+  - --device=dri
+  - --filesystem=host
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+
+modules:
+  - name: Protodesk
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/nemuelw/protodesk.git
+        tag: v1.2.7
+
+    build-commands:
+      - install -Dm644 flathub/io.github.nemuelw.protodesk.desktop /app/share/applications/io.github.nemuelw.protodesk.desktop
+      - install -Dm644 assets/logo.svg /app/share/icons/hicolor/scalable/apps/io.github.nemuelw.protodesk.svg
+      - rm /app/share/metainfo/io.qt.PySide.BaseApp.metainfo.xml
+      - install -Dm644 flathub/io.github.nemuelw.protodesk.metainfo.xml /app/share/metainfo/io.github.nemuelw.protodesk.metainfo.xml
+      - cp -r ./assets /app/bin
+      - install -Dm755 app.py /app/bin/app.py
+      - echo '#!/bin/sh' > /app/bin/protodesk
+      - echo 'python3 /app/bin/app.py' >> /app/bin/protodesk
+      - chmod +x /app/bin/protodesk
```

Comment:
use a script source https://docs.flatpak.org/en/latest/module-sources.html#shell-and-script-sources